static/app/views/insights/sessions/components/insightLayoutContext.tsx

```diff
@@ -0,0 +1,68 @@
+import type {ReactNode} from 'react';
+import {createContext, useCallback, useContext, useEffect, useState} from 'react';
+
+import localStorage from 'sentry/utils/localStorage';
+import type {DomainView} from 'sentry/views/insights/pages/useFilters';
+import {
+  type CHART_MAP,
+  DEFAULT_LAYOUTS,
+  PAGE_CHART_OPTIONS,
+} from 'sentry/views/insights/sessions/components/settings';
+
+type TChart = keyof typeof CHART_MAP;
+type TCharts = readonly TChart[];
+interface TContext {
```

Comment:
Prefixing types with `T` really tripped me up, that's usually reserved for generics, no? Maybe something like `ChartPlacementOption` would be more descriptive here

Reply:
idk if a T prefix is related to generics... but I inlined the types and defaultContext values. no more names.